utility/rpm_publish/python-storops.spec

```diff
@@ -126,6 +127,20 @@ Library for managing Unity/VNX systems. Please refer to https://github.com/emc-o
 %exclude %{python3_sitelib}/storops_test*
 %endif
 
+
+%post -n python2-%{pypi_name}
+if [ "$1" -ge 1 ] ; then
+    echo NOTE: If need to managing VNX system, please install Unisphere CLI rpm package first. Contact with Dell/EMC support on that.
```

Comment:
@Murray-LIANG 
RPM installation might be unattended (e.g. via Satellite) so output from %post scriptlets would be ignored.
Better would be if you would create explicit subpackage python-storops-vnx which requires both python-storops and Unisphere rpm package and document to install that for VNX systems.
That will make missing Unisphere package always clearly visible.